test/ClangImporter/ctypes_parse.swift

```diff
@@ -9,6 +9,7 @@ func checkEquatablePattern(_ c: Color) {
     case red: return
     case green: return
     case blue: return
+    default: return
```

Comment:
I think so, but at the same time it's also something that the old check wasn't catching but should have - and there's a trivial migration path here.